### PR TITLE
Korean language support added

### PR DESCRIPTION
**Pull request by psychoet**
kr.js file has been uploaded on lnbits/lnbits/static/i18n folder.

### CODE REVIEW

File: lnbits/static/i18n/kr.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
+window.localisation.en = {
```
Comment (prusnak) [inline lnbits/static/i18n/kr.js:1]:
```suggestion
window.localisation.kr = {
```

File: lnbits/static/i18n/kr.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  update_currency: '통화 수정하기',
+  press_to_claim: '비트코인을 수령하려면 눌러주세요',
+  donate: '기부',
+  view_github: 'GitHub 페이지 보기',
+  voidwallet_active: 'VoidWallet이 활성화되었습니다! 결제가 불가능합니다.',
+  use_with_caution: '주의하세요 - %{name} 지갑은 아직 BETA 단계입니다.',
+  service_fee: '서비스 수수료: 거래액의 %{amount} %',
+  service_fee_max: '서비스 수수료: 거래액의 %{amount} % (최대 %{max} sats)',
+  service_fee_tooltip:
+    'Service fee charged by the LNbits server admin per outgoing transaction 지불 결제 시마다 LNBits 서버 관리자에게 납부되는 서비스 수수료',
```
Comment (prusnak) [inline lnbits/static/i18n/kr.js:60]: Why is the first part of the string untranslated?

**Review by prusnak — CHANGES_REQUESTED** [review]

File: lnbits/static/i18n/kr.js (hunk 2 above)
Comment (psychoet) [reply lnbits/static/i18n/kr.js:60]: I missed the first line. Thanks!

**Comment by prusnak** [comment]
I reworked your PR to PR https://github.com/lnbits/lnbits/pull/2119 - it adds more stuff that need to be added in order to add Korean into the user interface. Thank you for your contribution to LNbits!

**Pull request closed without merging**